platform/administer/connector/database.mdx

```diff
@@ -2,37 +2,42 @@
 title: Database Connector
 sidebar_label: Database
 sidebar_position: 1
-description: Learn how to use Database Connectors with vCluster
+description: Learn how to use Database Connectors with the platform to manage backing stores for multiple virtual clusters.
 ---
 
-The database connector feature allows an admin to configure a database server to be used to manage backing stores for multiple virtual clusters. For each virtual cluster, the feature automatically creates individual databases and
-creates a non-privileged user. Read more about vCluster's using of external
-databases in the [vCluster External Database docs](/docs/vcluster/configure/vcluster-yaml/control-plane/components/backing-store/database/external).
+import BasePrerequisites from '../../_partials/install/base-prerequisites.mdx';
 
+import InstallNextSteps from '../../_partials/install/install-next-steps.mdx';
+
+The database connector feature allows an admin to configure a database server to manage backing stores for multiple virtual clusters. For each virtual cluster, the feature automatically creates individual databases and non-privileged users. Read more about the use of external databases in the [External Database documentation](/docs/vcluster/configure/vcluster-yaml/control-plane/components/backing-store/database/external).
```

Comment:
`the feature automatically creates individual databases and non-privileged users` should be `the feature automatically creates an independent database and non-privileged user`. Each virtual cluster only gets one of these.